keras_rcnn/preprocessing/_object_detection.py

```diff
@@ -1,118 +1,84 @@
-import threading
-
+import keras.backend
+import keras.preprocessing.image
 import numpy
-import numpy.random
+import skimage.transform
 import skimage.io
-import keras.utils
-import sklearn.preprocessing
-
-
-class Iterator:
-    def __init__(self, n, batch_size, shuffle, seed):
-        self.batch_index = 0
-
-        self.batch_size = batch_size
-
-        self.index_generator = self._flow_index(n, batch_size, shuffle, seed)
-
-        self.lock = threading.Lock()
-
-        self.n = n
-
-        self.shuffle = shuffle
-
-        self.total_batches_seen = 0
-
-    def reset(self):
-        self.batch_index = 0
-
-    def _flow_index(self, n, batch_size=32, shuffle=False, seed=None):
-        self.reset()
-
-        while True:
-            if seed is not None:
-                numpy.random.seed(seed + self.total_batches_seen)
-
-            if self.batch_index == 0:
-                index_array = numpy.arange(n)
-
-                if shuffle:
-                    index_array = numpy.random.permutation(n)
+import time
 
-            current_index = (self.batch_index * batch_size) % n
+def scale_shape(shape, min_size=224, max_size=224):
```

Comment:
Sure, I can change that. But I think it would be useful to agree on whether we are going to use (width, height) tuples, from now on, or (height, width). I saw different usage in different parts of the code.